subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/AndroidTestProject.groovy

```diff
@@ -72,6 +78,20 @@ class IncrementalAndroidTestProject extends AndroidTestProject {
     String pathToChange
     String taskToRunForChange
 
+    void configureForLatestAgpVersionOfMinor(GradleProfilerCrossVersionPerformanceTestRunner runner, String lowerBound) {
+        runner.addBuildMutator { invocationSettings ->
+            new BuildMutator() {
+                @Override
+                void beforeScenario() {
+                    AGP_VERSIONS.replaceAgpVersion(
```

Comment:
I would much prefer to use a system property which is passed to the build (and used in `pluginManagement`) and not editing the build script. Note that if you don't revert the edit in `afterScenario`, then the next build will pick up the same version again.

Reply:
Good point, I changed that